tests/scripts/all.sh

```diff
@@ -2365,48 +2201,60 @@ component_test_psa_crypto_config_accel_ecjpake_use_psa () {
     scripts/config.py -f include/psa/crypto_config.h unset PSA_WANT_ALG_ECB_NO_PADDING
 
     loc_accel_flags=$( echo "$loc_accel_list" | sed 's/[^ ]* */-DLIBTESTDRIVER1_MBEDTLS_PSA_ACCEL_&/g' )
-    make -C tests libtestdriver1.a CFLAGS=" $ASAN_CFLAGS $loc_accel_flags" LDFLAGS="$ASAN_CFLAGS"
+    # SHA-1 and all variants of SHA-2 are needed for ECDSA and X.509 tests.
+    # Note = we are NOT adding these ALG_SHA_xxx to the "loc_accel_flags" list
```

Comment:
I'm sorry, but this comment is really hard to read, even though I understand what's the reason behind it. This is probably because of the double negation that reads strange. I would suggest spliting this sentence and explaining that `loc_accel_list` is accelerated by both the test driver library and the main library (by running both sed commands on this variable), and that we want SHA_XXX only to be accelerated on the driver side to allow the driver code to use it, but not to use the SHA_XXX drivers directly by the library.

Reply:
Note: I intend to rework this at some point in the (hopefully near) future, as we're starting to have quite a lot of duplicated code and comments between the various `accel` components. When I do, I'll try to make this clearer at the same time. (But since I don't have a timeframe for that, we should still try to make things readable in the meantime.)

One thing we could do that might improve the situation a bit without requiring too many changes to replace this code block with something like:
```
# Things we wanted supported in libtestdriver1, but not accelerated in the main library:
# SHA-1 and all SHA-2 variants, as they are used by ECDSA deterministic.
loc_extra_list="ALG_SHA1 ALG_SHA_224 ALG_SHA_256 ALG_SHA_384 ALG_SHA_512"
loc_accel_flags=$( echo "$loc_accel_list $loc_extra_list" | sed 's/[^ ]* */-DLIBTESTDRIVER1_MBEDTLS_PSA_ACCEL_&/g' )
make ... # [as before]
```

Wdyt?

Reply:
`# Things we want accelerated in libtestdriver1, but not in the main library:` sounds a bit better in my opinion, but doesn't convey all the information that the original comment had. 
I would say that this is minor, since your're planning to rework it anyway, so let's not focus too much on that.